lib/xcodeproj/project/object/target_dependency.rb

```diff
@@ -39,6 +39,16 @@ def display_name
           return target_proxy.remote_info if target_proxy
         end
 
+        # @return [String] uuid of the target, if the dependency
+        #         is a native target, otherwise the uuid of the
+        #         target in the sub-project if the dependency is
+        #         a target proxy
+        #
+        def native_target_uuid
+          return target.uuid if target
+          return target_proxy.remote_global_id_string if target_proxy
```

Comment:
This is actually fixed in the line below, but it didn't close this comment since I didn't change these lines :)